### ISSUE TITLE
Nothing is recorded after the request phase

### ISSUE THREAD

**Issue by luluxing** [thu-pacman/LiveGraph-Binary#2]
Hello! I followed the instructions to reproduce the linkBench result, but after load and request, nothing is saved to record/ directory.

Here is how I did:
1. clone the LinkBench repo
2. download the FacebookLinkBench.jar and put it in target/ directory
3. start MySQL server and create the table schemas
4. modify the configuration files and change maxid1 to be 10001
5. run `./bin/linkbench -c some.properties -l` to load the data
6. run `./bin/linkbench -c some.properties -r` to request

The request phase completed successfully, but the traces are not recorded. Thanks in advance!

**Comment by saffahyjp** [comment]
Hi, thank you for opening the issue. I'm working to figure out what might be happening.

**Comment by saffahyjp** [comment]
Have you tried to use both -l and -r in a single run? Currently we don't support separate runs of -l and -r

**Comment by luluxing** [comment]
Yes, I tried with `./bin/linkbench -c some.properties -l -r` but didn't find anything in `pwd/record/`.

**Comment by rwang067** [comment]
> Yes, I tried with `./bin/linkbench -c some.properties -l -r` but didn't find anything in `pwd/record/`.

Hi, is this issues solved? I also encountered the same problem.

**Comment by saffahyjp** [comment]
Hi, sorry for a late reply. I have reproduced the problem and I am trying to solve it. Sorry...

**Comment by rwang067** [comment]
> Hi, sorry for a late reply. I have reproduced the problem and I am trying to solve it. Sorry...

Thanks  a lot and I'm looking forward to your reply.

**Comment by saffahyjp** [comment]
Hi, I have updated `README.md`, mainly with the following changes:

1. In your config file (`config/Something.properties`), change the lines to be `linkstore = com.facebook.LinkBench.GraphStoreWrapper` and `nodestore = com.facebook.LinkBench.GraphStoreWrapper`.
2. Before running the driver, make sure there exists an empty directory named `record` in the current working directory.

I believe this would solve the problem. Sorry again for the inconvenience caused.

**Comment by rwang067** [comment]
> Hi, I have updated `README.md`, mainly with the following changes:
> 
> 1. In your config file (`config/Something.properties`), change the lines to be `linkstore = com.facebook.LinkBench.GraphStoreWrapper` and `nodestore = com.facebook.LinkBench.GraphStoreWrapper`.
> 2. Before running the driver, make sure there exists an empty directory named `record` in the current working directory.
> 
> I believe this would solve the problem. Sorry again for the inconvenience caused.

It works! Thank you so much.
